libs/feature/catalog/src/lib/organisations/service/organisations-from-metadata.service.spec.ts

```diff
@@ -317,23 +317,19 @@ describe('OrganisationsFromMetadataService', () => {
         title: 'Surval - Données par paramètre',
         uuid: 'cf5048f6-5bbf-4e44-ba74-e6f429af51ea',
         contact: {
-          name: 'Ifremer',
-          email: 'ifremer.ifremer@ifremer.admin.ch',
-          logoUrl: 'http://localhost/geonetwork/images/harvesting/ifremer.png',
+          name: "Cellule d'administration Quadrige",
+          organisation: 'Ifremer',
+          email: 'q2suppor@ifremer.fr',
+          website: 'https://www.ifremer.fr/',
+          logoUrl:
+            'http://localhost/geonetwork/images/logos/81e8a591-7815-4d2f-a7da-5673192e74c9.png',
         },
         resourceContacts: [
           {
             email: 'ifremer.ifremer@ifremer.admin.ch',
-            logoUrl:
-              'http://localhost/geonetwork/images/harvesting/ifremer.png',
-            name: 'Ifremer',
-            organisation: 'Ifremer',
-          },
-          {
-            email: 'q2_support@ifremer.fr',
             logoUrl:
```

Comment:
The test is wrong.
You should write what you expect first, and code the correct behavior, instead of coding and adapting the test so it can pass :wink: 

Here what we said
1. logo of the contact if it exists
2. logo of the organisation
3. logo of the record

In your test, it set the logo of the record while it should use the logo of the organisation instead (because `resourceContact[0].logoUrl = ''`)

Probably, during the mapping, the resource contact logoUrl is set to the record logo if null in the contact, before the mapping with the organisation, which breaks the priority.